src/modules/flow/iio/nodes.c

```diff
@@ -275,8 +275,10 @@ gyroscope_open(struct sol_flow_node *node, void *data, const struct sol_flow_nod
 
     mdata->iio_base.config.buffer_size = opts->buffer_size;
     mdata->iio_base.config.sampling_frequency = opts->sampling_frequency;
-    snprintf(mdata->iio_base.config.sampling_frequency_name,
+    ret = snprintf(mdata->iio_base.config.sampling_frequency_name,
```

Comment:
After check snprintf manual. ret is  Upon successful return, these functions return the number of characters printed (excluding the null byte used to end output to strings).

For e.g 
"in_anglvel_sampling_frequency" is 29 bytes.
sizeof(mdata->iio_base.config.sampling_frequency_name) is NAME_MAX. I think is 256 bytes
29 >= NAME_MAX which is wrong will go to err (Line 280)